shared/constants/metametrics.ts

```diff
@@ -638,6 +638,7 @@ export enum MetaMetricsEventName {
   AccountDetailMenuOpened = 'Account Details Menu Opened',
   BlockExplorerLinkClicked = 'Block Explorer Clicked',
   AccountRemoved = 'Account Removed',
+  AccountRemoveFailed = 'Account Remove Failed',
```

Comment:
Why is this event outside the `keyring-snap` fence?

Reply:
Well, I thought this one could be used in a "generic way" for other accounts. So indeed, for now, it's only used in the context of snap accounts. Having a `SnapAccountRemoved` (or similar) could make it harder if we decide to update the event name later on.

But I can definitely put it inside fence and rename it too. WDYT?

Reply:
I'm ok with leaving it like that, just wanted to know the reason behind it